osu.Game/Beatmaps/BeatmapInfo.cs

```diff
@@ -93,6 +93,11 @@ public class BeatmapInfo : IEquatable<BeatmapInfo>, IHasPrimaryKey
         public bool WidescreenStoryboard { get; set; }
         public bool EpilepsyWarning { get; set; }
 
+        /// <summary>
+        /// Whether or not sound samples should change rate when playing with speed-changing mods.
```

Comment:
Probably worth mentioning somewhere around here that this is `TODO: requires implementation in gameplay`